### PR TITLE
Notification fix

### PR DESCRIPTION
**Pull request by iravisankar**

### CODE REVIEW

File: ios/App/App.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		504EC3131FED79650016851F /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		50B271D01FEDC1A000F3C39B /* public */ = {isa = PBXFileReference; lastKnownFileType = folder; path = public; sourceTree = SOURCE_ROOT; };
 		51235D8B26BBDDF200CA7747 /* GoogleService-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist; path = "GoogleService-Info.plist"; sourceTree = "<group>"; };
+		5150334C26CC29370016E2BB /* AppRelease.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; path = AppRelease.entitlements; sourceTree = "<group>"; };
```
Comment (iamtarun1993) [inline ios/App/App.xcodeproj/project.pbxproj:32]: @Dimple16 can you check this is the same in your prod app release folder?

File: ios/App/App.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -76,6 +77,7 @@
 		504EC3061FED79650016851F /* App */ = {
 			isa = PBXGroup;
 			children = (
+				5150334C26CC29370016E2BB /* AppRelease.entitlements */,
```
Comment (iamtarun1993) [inline ios/App/App.xcodeproj/project.pbxproj:80]: Check this also @Dimple16

File: ios/App/App.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -389,14 +391,14 @@
 			baseConfigurationReference = AF51FD2D460BCFE21FA515B2 /* Pods-App.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_ENTITLEMENTS = App/App.entitlements;
+				CODE_SIGN_ENTITLEMENTS = App/AppRelease.entitlements;
```
Comment (iamtarun1993) [inline ios/App/App.xcodeproj/project.pbxproj:394]: same here @Dimple16

File: ios/App/App/AppRelease.entitlements [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iamtarun1993) [inline ios/App/App/AppRelease.entitlements:1]: most probably we need this file also..but good to check
@Dimple16

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**